iid/iid.go

```diff
@@ -0,0 +1,75 @@
+// Copyright 2017 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package iid contains functions for deleting instance IDs from Firebase projects.
+package iid
+
+import (
+	"errors"
+	"fmt"
+	"net/http"
+
+	"google.golang.org/api/transport"
+
+	"firebase.google.com/go/internal"
+
+	"golang.org/x/net/context"
+)
+
+const iidEndpoint = "https://console.firebase.google.com/v1"
+
+// Client is the interface for the Firebase instance ID service.
+type Client struct {
+	endpoint string
```

Comment:
Maybe add a comment that this is here only for replacing it during testing?

Reply:
Done